mailbox/api/src/test/java/org/apache/james/mailbox/model/MailboxPathTest.java

```diff
@@ -220,6 +248,12 @@ void hasEmptyNameInHierarchyShouldBeTrueIfPathWithAnEmptyNameBetweenTwoNames() {
             .hasEmptyNameInHierarchy('.'))
             .isTrue();
     }
+    @Test
+    void hasEmptyNameInHierarchyShouldBeTrueIfPathWithAaBlankNameBetweenTwoNames() {
```

Comment:
```suggestion
    void hasEmptyNameInHierarchyShouldBeTrueIfPathWithABlankNameBetweenTwoNames() {
```
Also skip a line before `@Test` tag